tests/config.c

```diff
@@ -28,9 +28,6 @@
 #define TEST_CONFIG_DEFAULT_MAX_BRANCHES_PER_NODE 24
 
 // Deal with reasonable key / message sizes for tests
-// There are open issues in some tests for smaller key-sizes.
-// For now, restrict tests to use this minimum key-size.
-#define TEST_CONFIG_MIN_KEY_SIZE         ((int)sizeof(uint64))
```

Comment:
Define moved to `tests/config.h` as it's now also referenced in `tests/unit/large_inserts_stress_test.c`.